src/util/cmdline.h

```diff
@@ -12,57 +12,36 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <list>
 #include <string>
 #include <vector>
+#include <boost/program_options.hpp>
 
-enum opt_types
+namespace po = boost::program_options;
+#include <iterator>
+// A helper function to print a vector.
+template <class T>
+std::ostream &operator<<(std::ostream &os, const std::vector<T> &v)
 {
-  switc,
-  number,
-  string
-};
-
-struct opt_templ
-{
-  char optchar;
-  std::string optstring;
-  opt_types type;
-  std::string init;
-};
+  copy(v.begin(), v.end(), std::ostream_iterator<T>(os, " "));
+  return os;
+}
```

Comment:
Why do you need this function in a header?

Reply:
For testing something, but forget to delete it.